tests/tests.py

```diff
@@ -350,10 +350,8 @@ def test_channels_list_moderator(self):
         me = self.rocket.me().json()
         channels_add_moderator = self.rocket.channels_add_moderator(
             'GENERAL', me.get('_id')).json()
-        print(channels_add_moderator)
         channels_list_moderator = self.rocket.channels_moderators(
             room_id='GENERAL').json()
-        print(channels_list_moderator)
         self.assertTrue(channels_list_moderator.get('success'))
         channels_remove_moderator = self.rocket.channels_remove_moderator(
```

Comment:
Same here. If you don't need these variables just remove them, but I would advise to just checking if the API call succeeded.